### PR TITLE
Timeout shutdown.

### PR DESCRIPTION
**Pull request by jeschkies**
JIRA issues: MARATHON-8592

Co-authored-by: Nikita Melkozerov <nmelkozerov@mesosphere.io>

### CODE REVIEW

**Comment by jeschkies** [comment]
@timcharper we still need to know why the crash is called twice from the leader election service.

**Comment by jeschkies** [comment]
Ping @timcharper

**Comment by jeschkies** [comment]
Ping @timcharper once more.

File: src/main/scala/mesosphere/marathon/MarathonConf.scala [hunk 1]
```diff
@@ -163,6 +163,11 @@ trait MarathonConf
     true
   }
 
+  lazy val gracefulShutdownTimeout = opt[Long] (
```
Comment (timcharper) [inline src/main/scala/mesosphere/marathon/MarathonConf.scala]: oof. Do we really need an option for this? Can we use an undocumented environment variable at first?

Comment (jeschkies) [reply src/main/scala/mesosphere/marathon/MarathonConf.scala]: Fair point.

**Pull request merged**